### PR TITLE
Enable qemu fallback

### PR DESCRIPTION
**Pull request by apatard**
This pull request aims at fixing the test suite:
- there's a new black error
- change CI configuration for the molecule-devel, to more or less match what's done in molecule
- switch to e1000 for the alpine box, in hope it'll solve the ssh time out issue in CI

### CODE REVIEW

File: tox.ini
Comment (ssbarnea) [inline tox.ini:80]: This is clearly a typo :D

Comment (apatard) [reply tox.ini:80]: oh. oops. I'll wait for the CI to finish and fix this annoying typo.

**Comment by apatard** [comment]
I've put some workaround for hosts without kvm (no /dev/kvm -> switch to tcg) but I'm getting a new error in the CI, so I've yet to know if the workaround is working in the CI. I'm going to wait for the CI to finish and try debugging it tomorrow.
I guess i should also add something in the logs to show if there's virtualisation or not.

File: tools/test-setup.sh
Comment (ssbarnea) [inline tools/test-setup.sh:28]: /bin/true is not portable, just "true"

Comment (apatard) [reply tools/test-setup.sh]: Fixed

**Comment by apatard** [comment]
recheck

**Review by ssbarnea — CHANGES_REQUESTED** [review]
Sadly I need to refuse this because it breaks on macos. See https://gist.github.com/ssbarnea/b65cc225c5cd279c9d1b36c35be98307

Code from master worked for me on macos, but when I run this patch it failed, so it would be a clear regression.

**Review by ssbarnea — APPROVED** [review]
I tried again and it failed, apparently in similar way so no reason to block it.

File: tox.ini
Comment (ssbarnea) [inline tox.ini:18]: Please remove these from the change, they are unrelated, ok for debugging but no reason to make it run spammy.

I think that this change should not touch the tox.ini file.

Comment (apatard) [reply tox.ini:18]: actually, molecule is setting it too. I've put it there to be closer to what's current practice in molecule's ``tox.ini``

**Comment by apatard** [comment]
Do you have more information about the regression ? Can it be a libvirt vs vbox thing ? It would be nice to see this fixed before merging.

**Comment by apatard** [comment]
After our discussion on irc, I thought about it and I think I'm going to split this PR to have at least two, at least, in order to have the qemu fallback stuff in its own PR.
I'm also going to have a different PR for your failure for VBox. The failing test can't work on vbox, since vbox doesn't support rtl8139* cards.

**Pull request merged**
